config/development.json

```diff
@@ -172,6 +172,7 @@
 		"woocommerce/extension-settings-payments": true,
 		"woocommerce/extension-settings-shipping": true,
 		"woocommerce/extension-settings-tax": true,
+		"woocommerce/extension-settings-email": true,
```

Comment:
Lets added entries into the other config files as well, setting those to false.

Reply:
Thanks Justin for the review. I will have a clean PR with what I wanted to do that will have proper commits used - the rever created new commit so now I will have potential merge problems with my other PRs that were forked from the PR that is origin of this PR.

Reply:
I will put your comments there and fix them at that PR.